### PR TITLE
Create resource.md for motion planning

### PR DESCRIPTION
**Pull request by patilunmesh**
added few resources.

### CODE REVIEW

**Comment by patilunmesh** [comment]
looks like the formatting is messed up in md.

**Comment by patilunmesh** [comment]
Done with formatting and also I have added some more resources

**Comment by take2rohit** [comment]
@patilunmesh rename file to `resources.md` to `README.md`. so that its rendered when we open the folder

File: Motion_planning/resource.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+1. https://www.edx.org/course/autonomous-mobile-robots
+2. https://www.coursera.org/learn/robotics-motion-planning
+
+Intermediate:
+
+1. https://www.coursera.org/learn/motion-planning-self-driving-cars
+2. 
+
+# books:
```
Comment (take2rohit) [inline Motion_planning/resource.md]: Capital b

File: Motion_planning/resource.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+#Courses:
+beginners:
```
Comment (take2rohit) [inline Motion_planning/resource.md]: Beginners capital b

**Review by take2rohit — CHANGES_REQUESTED** [review]

File: Motion_planning/resource.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Resources for motion planning
+
+## Courses:
+### beginners:
```
Comment (akshaykvnit) [inline Motion_planning/resource.md:4]: Capital B? Make similar changes where first letter is not capitalized (like books)

File: Motion_planning/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+### Intermediate:
+
+1. https://www.coursera.org/learn/motion-planning-self-driving-cars
+
+## Books:
+### Introductory:
+1.  [Robot motion planning intro](http://ais.informatik.uni-freiburg.de/teaching/ss11/robotics/slides/18-robot-motion-planning.pdf)
+
+### Full concept:
```
Comment (akshaykulkarni07) [inline Motion_planning/README.md]: please use something better than 'full concept' 😅

File: Motion_planning/README.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+0. [Principles of robot motion](http://biorobotics.ri.cmu.edu/book/)
```
Comment (akshaykulkarni07) [inline Motion_planning/README.md]: Use consistent numbering (1-indexed, not 0-indexed).

**Comment by take2rohit** [comment]
I think we can merge resources without many reviews as people adding here must have added after some experience only

File: Motion_planning/README.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+### Intermediate:
+
+1. https://www.coursera.org/learn/motion-planning-self-driving-cars
+
+## Books:
+1. [Robot motion planning intro](http://ais.informatik.uni-freiburg.de/teaching/ss11/robotics/slides/18-robot-motion-planning.pdf)
+2. [Principles of robot motion](http://biorobotics.ri.cmu.edu/book/)
+3. [Planning Algorithms by S. M. Lavalle](http://planning.cs.uiuc.edu/booka4.pdf)
+4. [ Probabilistic robotics by S. Thrun](https://docs.ufpr.br/~danielsantos/ProbabilisticRobotics.pdf)
```
Comment (take2rohit) [inline Motion_planning/README.md]: `[ P` extra space

File: Motion_planning/README.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+## Courses:
+### Beginners:
+
+1. https://www.edx.org/course/autonomous-mobile-robots
+2. https://www.coursera.org/learn/robotics-motion-planning
+
+### Intermediate:
+
+1. https://www.coursera.org/learn/motion-planning-self-driving-cars
```
Comment (take2rohit) [inline Motion_planning/README.md:11]: Please make in this format `[Robot motion planning intro](http://ais.informatik.uni-freiburg.de/teaching/ss11/robotics/slides/18-robot-motion-planning.pdf)`

File: Motion_planning/README.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+### Path planning Thesis:
+1. [KTH karl kurzer](https://www.researchgate.net/profile/Karl_Kurzer/publication/337719881_Path_Planning_in_Unstructured_Environments_A_Real-time_Hybrid_A_Implementation_for_Fast_and_Deterministic_Path_Generation_for_the_KTH_Research_Concept_Vehicle/links/5de6b10292851c83645fc109/Path-Planning-in-Unstructured-Environments-A-Real-time-Hybrid-A-Implementation-for-Fast-and-Deterministic-Path-Generation-for-the-KTH-Research-Concept-Vehicle.pdf)
+
+## Important libraries and webpages:
+1. [OMPL](https://ompl.kavrakilab.org/)
+2. [RRT](http://msl.cs.uiuc.edu/rrt/)
+3. [CMU motion planning course homepage](http://www.cs.cmu.edu/~./motionplanning/)
+
+## Important publications and research groups:
+1. http://ai.stanford.edu/~latombe/pub.htm
```
Comment (take2rohit) [inline Motion_planning/README.md:30]: please make in this format `[RRT](http://msl.cs.uiuc.edu/rrt/)`

**Review by take2rohit — CHANGES_REQUESTED** [review]
Please do the changes

File: Motion_planning/README.md (hunk 8 above)
Comment (patilunmesh) [reply Motion_planning/README.md:30]: nope

Comment (akshaykulkarni07) [reply Motion_planning/README.md:30]: @patilunmesh please be a little serious and put some effort. This is your initiative after all. It should look good.

**Comment by patilunmesh** [comment]
Please stop taking out ignorable typos. Add content if any. I don't think those small spaces really matter here. @take2rohit

File: Motion_planning/README.md (hunk 7 above)
Comment (patilunmesh) [reply Motion_planning/README.md:11]: All these links are self-explanatory that's why they are not hyperlinked.

**Comment by akshaykulkarni07** [comment]
They matter. Fix it in the next commit, along with content if someone suggests something. Also, personal message some senior to review these things. Assigning on github hasn't worked.
@patilunmesh also, please take criticism constructively and avoid those mistakes next time 😅.

**Comment by patilunmesh** [comment]
@akshaykvnit  I have fixed those already

**Comment by akshaykulkarni07** [comment]
All right, since you won't change, I'm merging the PR. Someone else can look at the content later.

**Pull request merged**